# HG changeset patch
# User ATAGOYAMA Tanuki <s172262@yahoo.co.jp>
# Date 1511265096 -32400
# Branch fix-python-error
# Node ID 50b39a5f4dcce2a46d0375fe976e3d04b268b1b7
# Parent  6ee62e8973a17877e27976760b19148e99e372e8
cegui/src/ScriptModules/Python/binding/CMakeLists.txt: fix cmake error

diff --git a/cegui/src/ScriptModules/Python/bindings/CMakeLists.txt b/cegui/src/ScriptModules/Python/bindings/CMakeLists.txt
--- a/cegui/src/ScriptModules/Python/bindings/CMakeLists.txt
+++ b/cegui/src/ScriptModules/Python/bindings/CMakeLists.txt
@@ -1,5 +1,5 @@
 execute_process(
-    COMMAND ${PYTHON_EXECUTABLE} -c "import distutils.sysconfig; print distutils.sysconfig.get_python_lib(plat_specific=True, prefix=\"\")"
+    COMMAND ${PYTHON_EXECUTABLE} -c "import distutils.sysconfig; print (distutils.sysconfig.get_python_lib(plat_specific=True, prefix=\"\"))"
     OUTPUT_VARIABLE CEGUI_PYTHON_INSTALL_DIR
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
